cns/client/client_test.go

```diff
@@ -124,8 +124,8 @@ func getIPNetFromResponse(resp *cns.IPConfigResponse) (net.IPNet, error) {
 	)
 
 	// set result ipconfig from CNS Response Body
-	prefix := strconv.Itoa(int(resp.PodIpInfo.PodIPConfig.PrefixLength))
-	ip, ipnet, err := net.ParseCIDR(resp.PodIpInfo.PodIPConfig.IPAddress + "/" + prefix)
+	prefix := strconv.Itoa(int(resp.PodIpInfo[0].PodIPConfig.PrefixLength))
```

Comment:
this code needs to be updated to have a loop that runs over PodIpInfo instead of hardcoding 0th index